include/mfmg/dealii/amge_host.templates.hpp

```diff
@@ -192,7 +192,8 @@ AMGe_host<dim, MeshEvaluator, VectorType>::compute_local_eigenvectors(
   {
     boost::property_tree::ptree lanczos_params;
     lanczos_params.put("num_eigenpairs", n_eigenvectors);
-    lanczos_params.put("tolerance", tolerance);
+    // lanczos_params.put("tolerance", tolerance);
+    lanczos_params.put("tolerance", 1e-6);
```

Comment:
Did you mean to commit that?

Reply:
Yes, but this is all in progress. I need to put a lot more comments about why I'm doing things this way.

Reply:
Please do so.